packages/shared/lib/sdk/sync.unit.test.ts

```diff
@@ -457,25 +457,26 @@ describe('Log', () => {
         await nangoAction.log('hello', { level: 'error' });
 
         expect(mock).toHaveBeenCalledWith({
-            data: {
-                activityLogId: '1',
-                log: {
-                    type: 'log',
-                    level: 'error',
-                    message: 'hello',
-                    createdAt: expect.any(String),
-                    environmentId: 1,
-                    meta: null,
-                    source: 'user'
-                }
-            },
+            data: expect.any(String),
             headers: {
                 Authorization: 'Bearer ***',
                 'Content-Type': 'application/json'
             },
             method: 'POST',
             url: '/environment/1/log'
         });
+        expect(JSON.parse((mock.mock.calls as any)[0][0].data)).toStrictEqual({
```

Comment:
`(mock.mock.calls as any)[0][0].data)` that's scary 😨

Reply:
yep, not a fan but since everything is stringified now there was no way to use partial match without accessing the value directly :/